js/search.js

```diff
@@ -892,6 +892,38 @@
 				abilitySet.push(['header', "Special Event Ability"]);
 				abilitySet.push(['ability', toId(template.abilities['S'])]);
 			}
+			if (format === 'almostanyability' || format === 'balancedhackmons' || format === 'bh') {
+				template = Tools.getTemplate(set.species);
+				var abilities = [];
+				if (template.isMega) {
+					if (format === 'almostanyability') abilitySet.unshift(['html', '<p>Will be <strong>' + Tools.escapeHTML(template.abilities['0']) + '</strong> after Mega Evolving.</p>']);
+					template = Tools.getTemplate(template.baseSpecies);
+				}
+				for (var i in BattleAbilities) {
+					if (BattleAbilities[i].isNonstandard) continue;
+					if (BattleAbilities[i].gen > this.gen) continue;
+					abilities.push(i);
+				}
+
+				abilities.sort();
+
+				var goodAbilities = [['header', "Abilities"]];
+				var poorAbilities = [['header', "Situational Abilities"]];
+				var badAbilities = [['header', "Unviable Abilities"]];
+				for (var i = 0; i < abilities.length; i++) {
+					var id = abilities[i];
+					var rating = BattleAbilities[id] && BattleAbilities[id].rating;
+					if (id === 'normalize' && (format === 'balancedhackmons' || format === 'bh')) rating = 3;
+					if (rating >= 3) {
+						goodAbilities.push(['ability', id]);
+					} else if (rating >= 2) {
+						poorAbilities.push(['ability', id]);
```

Comment:
Is there a reason you're excluding ratings 0.5 to 1.5 in this list?

Are they straight outclassed or something?

Reply:
i looked over them all and everything with a 1.5 rating or lower was just bad

Reply:
especially in AAA and BH

Reply:
Do you have examples?